ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -10,4 +10,5 @@ export AZURE_AUTH_LOCATION=$cluster_profile/osServicePrincipal.json
 
 echo "Deprovisioning cluster ..."
 cp -ar "${SHARED_DIR}" /tmp/installer
-openshift-install --dir /tmp/installer destroy cluster
+TF_LOG=debug openshift-install --dir /tmp/installer destroy cluster
```

Comment:
destroy doesn't use terraform under the hood... So no need for debug tf_log

Reply:
amended. also added 06c8d90203416468421c9d4938ef04ef18aea039 to remove from vcenter.